include/mega/megaclient.h

```diff
@@ -1069,6 +1069,21 @@ class MEGA_API MegaClient
     Node* nodebyfingerprint(FileFingerprint*);
     node_vector *nodesbyfingerprint(FileFingerprint* fingerprint);
 
+    node_vector getRecentNodes(unsigned maxcount, m_time_t since, bool includerubbishbin);
+
+    struct recentaction
+    {
+        m_time_t time;
+        handle user;
+        handle parent;
+        bool updated;
+        bool media;
+        node_vector v;
+    };
+    typedef vector<recentaction> recentactions_vector;
```

Comment:
Definition of types usually go to the `types.h` file

Reply:
thanks, I wasn't aware of that convention, I'll follow it in future.  cheers